vyper/parser/external_call.py

```diff
@@ -37,8 +37,7 @@ def external_call(node, context, interface_name, contract_address, pos, value=No
     ]
     if context.is_constant() and not sig.const:
         raise ConstancyViolation(
```

Comment:
Is the context usage changed now where this is a broader error?

Reply:
Hrmm..  I thought it was still necessary, but it shows as not covered on codecov so probably can go.